libc/include/errno.h.def

```diff
@@ -44,7 +44,15 @@
 #endif
 
 #if !defined(__AMDGPU__) && !defined(__NVPTX__)
+
+#ifdef __cplusplus
+extern "C" {
+  extern thread_local int __llvmlibc_errno;
```

Comment:
libc/src/errno/libc_errno.cpp line 18 doesn't use indentation here; I think we should remove it here, too.